comit/src/expiries.rs

```diff
@@ -70,6 +70,18 @@ pub fn to_timestamps(
 /// always refers to the time used by the op codes used in the COMIT contracts.
 #[async_trait]
 pub trait CurrentTime {
+    // Two types that implement this trait do not necessarily support total
```

Comment:
Should we use a different struct that `Timestamp` here and **not** impl `Ord` or `PartialOrd` on it?